test/Cloudy/runtests.jl

```diff
@@ -161,7 +162,7 @@ white = Noise(log(2.0))
 # construct kernel
 GPkernel =  kern1 + kern2 + white
 # Get training points    
-u_tp, g_tp = Utilities.extract_GP_tp(ekiobj, N_iter)
+u_tp, g_tp = Utilities.extract_GP_tp(ekiobj, N_iter-1)
```

Comment:
Was this just a typo from before?

Reply:
The second argument to extract_GP_tp just defines the number of EKI layers/iterations to train on - if it's set to N_iter, the initial ensemble is included in the training points, and if it's set to N_iter-1, the initial ensemble is not included in the training points. There is no "right" or "wrong" number of iterations to use though, so either is fine.